internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization_test.go

```diff
@@ -155,15 +155,15 @@ func checkDestroy(s *terraform.State) error {
 			continue
 		}
 		ids := conversion.DecodeStateID(rs.Primary.ID)
-		roles, _, err := acc.Conn().CloudProviderAccess.ListRoles(context.Background(), ids["project_id"])
+		roles, _, err := acc.ConnV2().CloudProviderAccessApi.ListCloudProviderAccessRoles(context.Background(), ids["project_id"]).Execute()
```

Comment:
Can we do the same check here as in the setup_test.go (but expecting NOT_FOUND instead of role)?
As this will never find the azure access role

Reply:
nice good catch